.travis.yml

```diff
@@ -45,14 +48,15 @@ install:
 
 before_script:
   - dask-scheduler &
+  - SCHEDULER_PID=$!
 
 # uncomment the following lines to override the default test script
 script:
 - if [[ -a .git/shallow ]]; then git fetch --unshallow; fi
 - julia -e 'Pkg.clone(pwd()); Pkg.build("DaskDistributedDispatcher"); Pkg.test("DaskDistributedDispatcher"; coverage=true)'
 
 after_script:
-  - python ./test/teardown.py
+  - kill $SCHEDULER_PID
```

Comment:
Change `kill` to `kill -s TERM` to be more explicit about the signal you are sending.